src/Carousel.vue

```diff
@@ -395,7 +395,10 @@ export default {
           });
         });
         if (this.$parent.$el) {
-          this.mutationObserver.observe(this.$parent.$el, config);
+          let carouselInnerElements = document.getElementsByClassName('VueCarousel-inner');
```

Comment:
If you'd like, this could be a computed property. Seems like relevant and reusable information for eventual use elsewhere in the carousel

Reply:
Yea it seems like it's relevant and reusable information but I totally don't feel in the position to change it into computed property and to make more changes in the code then it's necessary to fix the bug.  Here wanted this mutation obeserver  to be set to each carousel  present within component.

Reply:
Sure thing!